packages/migrations/src/batched-migrations/batched-migration-runner.ts

```diff
@@ -177,6 +177,7 @@ export class BatchedMigrationRunner {
       }
 
       if (error) {
+        logger.error('Error running batched migration job', error);
```

Comment:
On second thought... let's give ourselves some more context?

```suggestion
        logger.error(`Error running job ${nextJob.id} for batched migration "${this.migration.filename}"`, error);
```

Reply:
Done in 68ef183d60ee2d2232ec7f7f11e42bc77f7e9db6.